packaging/scripts/create-svn-repositories

```diff
@@ -1,24 +0,0 @@
-#!/bin/bash
-
-APP_NAME="_APP_NAME_"
-CLI="${APP_NAME}"
-APP_HOME="$(${CLI} config:get APP_HOME)"
-SVN_REPOSITORIES="$(${CLI} config:get SVN_REPOSITORIES)"
-PORT=$(${CLI} config:get PORT)
-SYS_API_KEY="$(${CLI} config:get SYS_API_KEY)"
-APP_GROUP="$(${CLI} config:get APP_GROUP)"
-SERVER_USER="$(${CLI} config:get SERVER_USER)"
-SERVER_HOSTNAME="$(${CLI} config:get SERVER_HOSTNAME)"
-SERVER_PROTOCOL="$(${CLI} config:get SERVER_PROTOCOL)"
-
-${APP_HOME}/bin/ruby ${APP_HOME}/extra/svn/reposman.rb \
-  --openproject-host "http://127.0.0.1:${PORT}" \
-  --owner "${SERVER_USER}" \
-  --group "${APP_GROUP}" \
-  --public-mode '2750' \
-  --private-mode '2750' \
-  --svn-dir "${SVN_REPOSITORIES}" \
-  --url "file://${SVN_REPOSITORIES}" \
-  --key "${SYS_API_KEY}" \
-  --scm Subversion \
-  --verbose
```

Comment:
@kgalli, @crohr 

There are some changes in the management of repositories. In short, reposman.rb is no longer required. Instead, repository creation is handled directly from within OpenProject.

This will also need to be reflected by the packaging. The adaptations required should encase:
* Configuring the repository options, please see:
 * https://community.openproject.org/work_packages/21050
 * https://github.com/opf/openproject/blob/release/4.3/config/configuration.yml.example#L227 
* If possible, remove the remains of the cron job configured on existing installations.
* If possible, call a rake task on existing installations to bring already configured installations under OpenProject management (Not of the utmost importance).